book_source/source/module-7-geospatial-data/lesson-20-dissolve-join/joins.md

```diff
@@ -5,19 +5,32 @@
     <iframe src="https://www.loom.com/embed/45449525974a471ca97929ea5b13d2fc?sharedAppSource=personal_library" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
 </div>
 
+---
+
 We're going to take a quick step away from geospatial data for the next flew slides, but this concept will be very relevant for the example we will be doing with geospatial data.  
 
 Let's go back to the world of CSV data that we would process with `pandas` . Suppose I had two different `DataFrame` shown below. The left one `tas` stores information about the TAs and `grading` on the right stores information about students and which TA will be grading them ( `ta_id` and `grader_id` will match if that TA is grading that student). Our final goal will be trying to find the number of students each TA will grade.  
 
-```{image} https://static.us.edusercontent.com/files/XefeN513eek9ajoDicHmXplz
-:alt: TODO
-:width: 758
-:align: center
-```
+## `tas`
```

Comment:
Minor: Do you think we can get these tables to show up side-by-side? There might be a way to hack this with the [sidebar directive](https://jupyterbook.org/content/layout.html#full-width-markdown-content) or if we include some CSS to make them appear on the same line.

Reply:
Added some CSS to make them appear on the same slide on wider screen sizes. Let me know if it looks okay (including the ones on the quiz slide)